src/tray/windows/SDL_tray.c

```diff
@@ -174,16 +173,7 @@ static wchar_t *convert_label(const char *in)
 
     *c2 = '\0';
 
-    int len_w = MultiByteToWideChar(CP_UTF8, 0, escaped, len + 1, NULL, 0);
-    wchar_t *out = (wchar_t *)SDL_malloc(len_w * sizeof(wchar_t));
-
-    if (!out) {
-        SDL_free(escaped);
-        return NULL;
-    }
-
-    MultiByteToWideChar(CP_UTF8, 0, escaped, -1, out, len_w);
-
+    wchar_t *out = (wchar_t *)WIN_UTF8ToString(escaped);
```

Comment:
Using the explicit `WIN_UTF8ToStringW` variant here is better. The `(wchar_t *)` cast is not needed.